Firebase/InAppMessaging/Flows/FIRIAMDisplayExecutor.m

```diff
@@ -41,6 +41,8 @@ @interface FIRIAMDisplayExecutor () <FIRInAppMessagingDisplayDelegate>
 @property(nonatomic) BOOL impressionRecorded;
 @property(nonatomic, nonnull, readonly) id<FIRIAMAnalyticsEventLogger> analyticsEventLogger;
 @property(nonatomic, nonnull, readonly) FIRIAMActionURLFollower *actionURLFollower;
+// Used for displaying the test on device message error alert.
+@property(nonatomic, strong) UIWindow *alertWindow;
```

Comment:
This needs `#import <UIKit/UIKit.h>` at the top - I believe it's only working now due to implicit imports.

Reply:
Done.